2-mandatory/2-financial-times.js

```diff
@@ -23,15 +50,45 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    const newarray=[];
+    const number=/[0-9]/;
```

Comment:
I see array like this for the first time, can you explain what it does ? I'm just curious 🙂

Reply:
it means that it is(number) integer and it is not a letter or sth else